.github/workflows/reusable-release.yml

```diff
@@ -332,21 +332,61 @@ jobs:
           sbt "sonatypeBundleRelease"
 
   github-release:
-    name: 🔒 GitHub Release
-    if: needs.init.outputs.release_type == 'FULL_MAIN_BRANCH'
-    needs:  [push-release-commit, sign]
+    name: 🔒 Update GitHub
+    needs:  [init, push-release-commit, sign]
     runs-on: ubuntu-latest
     permissions:
       contents: write
+      pull-requests: write
```

Comment:
In order for the workflow to comment on PRs (notifying that a preview has been published), we need the [`pull-requests: write`](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#:~:text=GitHub%20Apps.%22-,pull%2Drequests,-Work%20with%20pull) permission.